notebooks/stress-tests.ipynb

```diff
@@ -38,8 +38,8 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #34.        :param target_fonction: Fonction to be executed by the threads
</pre><p>Out of sync gain between param doc and the actual function.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>